docs/bigquery-usage.rst

```diff
@@ -283,20 +279,22 @@ Run a query which can be expected to complete within bounded time:
    >>> query = """\
    SELECT count(*) AS age_count FROM dataset_name.person_ages
    """
-   >>> results = client.run_sync_query(query, timeout_ms=1000)
+   >>> job = client.run_sync_query(query)
+   >>> job.timeout_ms = 1000
```

Comment:
It doesn't feel like a win to have constructors w/ 10+ arguments for all settable properties:  rather, constructors should take the minimum needed to use the object, and optional config is done only via property setters.